# Vehicle information required to register to parking management system

VehicleIdentificationMsg identifiers

VehicleDimensionsMsg dimensions

VehicleTypeMsg type

ParkPreferencesMsg park_preferences

time entry_time
time pickup_time

VehicleStatusMsg status
